test/test_manifest.json

```diff
@@ -1599,6 +1599,16 @@
        "lastPage": 6,
        "type": "eq"
     },
+    {  "id": "issue4872",
```

Comment:
But, there's already a test with this id. Please keep in mind that every test must have a *unique* id, since otherwise you'd overwrite the existing reference images.

Hence this needs to be adjusted, e.g. to `"id": "issue4872-forms"` to prevent errors.
Also, to simplify things when looking at the `test_manifest.json` file, please place this new entry *just after* https://github.com/mozilla/pdf.js/blob/2962f4a2c5724466901428a10f4021e5047634bb/test/test_manifest.json#L1611

Reply:
About id: that was my guess.
Thx